$OpenBSD: patch-src_SDL_c,v 1.2 2019/01/03 17:55:08 thfr Exp $

disable breaking if no haptic feedback support which disallows gamecontroller API
pretend to be Linux in SDL_GetPlatform to allow compatibility with FNA games
until OpenBSD support has been rolled out (estimated end of 2018)

Index: src/SDL.c
--- src/SDL.c.orig
+++ src/SDL.c
@@ -228,8 +228,6 @@ SDL_InitSubSystem(Uint32 flags)
             }
         }
         SDL_PrivateSubsystemRefCountIncr(SDL_INIT_HAPTIC);
-#else
-        return SDL_SetError("SDL not built with haptic (force feedback) support");
 #endif
     }
 
@@ -449,7 +447,7 @@ SDL_GetPlatform()
 #elif __NETBSD__
     return "NetBSD";
 #elif __OPENBSD__
-    return "OpenBSD";
+    return "Linux";
 #elif __OS2__
     return "OS/2";
 #elif __OSF__
